source/cl/test/UnitCL/source/ktst_precision.cpp

```diff
@@ -413,6 +413,17 @@ UCL_EXECUTION_TEST_SUITE_P(HalfOperatorTest, testing::Values(OPENCL_C),
 
 using HalfMathBuiltins = HalfParamExecution;
 
+struct HalfMathBuiltinsPow : HalfMathBuiltins {
+  const std::vector<cl_ushort> &GetEdgeCases() const override {
+    static std::vector<cl_ushort> EdgeCases = [&] {
```

Comment:
How come this is done in a lambda?

Reply:
presumably to do it all in one static variable line.

Reply:
Yes, this is to directly construct `EdgeCases` as the vector we want, rather than first constructing it and then adding an element, because this way we are guaranteed the initialization happens exactly once without needing extra guards.

Reply:
Fair enough, yeah.